recycler/src/main/java/com/elpassion/android/commons/recycler/items/ItemAdapter.kt

```diff
@@ -1,4 +1,4 @@
-package com.elpassion.android.commons.recycler
+package com.elpassion.android.commons.recycler.items
```

Comment:
Having little item adapters for every item looks like a bit wired idea.
If we at least abstract away all android dependencies then it would make a lot more sense for me.